client/verta/verta/client.py

```diff
@@ -660,27 +662,29 @@ def find_datasets(self,
         list of :class:`Dataset`
 
         """
+        datasets = Datasets(self._conn, self._conf)
+        if dataset_ids:
+            datasets._msg.dataset_ids.extend(_utils.as_list_of_str(dataset_ids))
+        if sort_key:
+            datasets.sort(sort_key, not ascending)
+        if workspace:
+            datasets.with_workspace(workspace)
```

Comment:
Are these don't in-place or do they return a new object? How does that compare with the similar ones in tracking?

Reply:
Oops! They should (and do) return new objects. This needs to be fixed here.